Agreements/Airport/TCNE_MC.txt

```diff
@@ -1 +1 @@
-COPX:*:*:MAYLA:EGMC:*:London TC NE:MCAPP_W:*:4000:HDG
```

Comment:
MAYLA at 4000ft is inside Thames airspace, so I've moved the COPX further North so that EuroScope predicts the traffic will descend into Southend's.